dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -71,25 +71,24 @@
   = f.number_field :step_speed
 
 .field.callouteditor
-  - (0...3).each do |x|
-    %label Callout #{x+1}
-
-    %div{class: "callouts#{x}"}
+  %div{id: "callout_template"}
+    %div{class: "callout_0"}
+      %label Callout
       %table
         %tr
-          %td Use callout?
+          %td Clear callout?
```

Comment:
It's not clear what this checkbox does.  Maybe add a button instead to remove this specific row?